src/OrchardCore/OrchardCore.DisplayManagement/PositionWrapper.cs

```diff
@@ -50,5 +51,24 @@ public ValueTask<IShape> AddAsync(object item, string position)
         {
             throw new System.NotImplementedException();
         }
+
+        public static PositionWrapper TryWrap(object value, string position)
+        {
+            if (value is PositionWrapper wrapper)
```

Comment:
`is IPositioned`?

Reply:
I used `PositionWrapper` because return value must be both `IPositioned` and `IShape`

Reply:
Are you suggesting to change to return `IShape`? like below?
```cs
 public static IShape TryWrap(object value, string position)
    {
        if (value is IShape && value is IPositioned wrapper )
        {
           ....
                           return wrapper as IShape;
        }
  }
```